### DISCUSSION TITLE
Read MCP3208 ADC via SPI

### DISCUSSION THREAD

**Discussion by mellow5** [arendst/Tasmota#11525] [Q&A] [answered]
I want to read and calculate some values from an ADC (MCP3208) via SPI and use tasmota as stable web interface to provide the data via MQTT.

Any knowledge out there about SPI and tasmota beside the listed displays?

**Comment by barbudor** [comment, marked as answer]
You are correct, in Tasmota SPI Is mainly used for displays
Only NRF24 and MFRC522 drivers are non-display drivers using SPI
But why looking into an additional ADC ? ADS1x15 is already implemented over I2C.

**Comment by Jason2866** [comment]
Prefered are i2c devices. If there is a good reason why a SPI device should be added it will be conisidered if a PR is contributed.
